packages/api-client/src/views/Request/Request.vue

```diff
@@ -67,7 +67,7 @@ watch(isNarrow, (narrow) => (showSideBar.value = !narrow))
  */
 const selectedSecuritySchemeUids = computed(
   () =>
-    (isReadOnly.value
+    (isReadOnly
```

Comment:
Is this not reactive anymore I guess?

Reply:
Correct. I couldn't think of any need for it to be reactive. Also it can also just be isModal

Reply:
Well we also have `layout = 'modal'` from [`useLayout`](https://github.com/scalar/scalar/blob/main/packages/api-client/src/hooks/useLayout.ts) we could just use that (also not reactive though).

Reply:
yea we probably should but I'll leave it for a follow up

Reply:
https://github.com/scalar/scalar/issues/3942